spiketoolkit/sorters/basesorter.py

```diff
@@ -58,10 +58,10 @@ def __init__(self, recording=None, output_folder=None, debug=False,
             # several groups
             self.recording_list = se.get_sub_extractors_by_property(recording, grouping_property)
             n_group = len(self.recording_list)
-            if n_group>1:
-                self.output_folders = [Path(str(output_folder) + '_'+str(i)) for i in range(n_group)]
+            if n_group > 1:
+                self.output_folders = [output_folder / str(i) for i in range(n_group)]
             else:
-                self.output_folders = [output_folder]
+                self.output_folders = [output_folder / str(0)]
```

Comment:
Does it break launcher ?

Reply:
Seems to pass the tests. I think we if the gather_comparisons.. can deal with recordings sorted by group. Let's keep this open, I will test tomorrow